security-jwt/src/main/java/io/micronaut/security/token/jwt/cookie/JwtCookieLoginHandler.java

```diff
@@ -80,6 +85,63 @@ public JwtCookieLoginHandler(RedirectConfiguration redirectConfiguration,
         super(jwtCookieConfiguration, redirectConfiguration, priorToLoginPersistence);
         this.accessTokenConfiguration = accessTokenConfiguration;
         this.accessRefreshTokenGenerator = accessRefreshTokenGenerator;
+        this.refreshTokenCookieConfiguration = createDefaultConfiguration(jwtCookieConfiguration);
+    }
+
+    /**
+     * @param redirectConfiguration Redirect configuration
+     * @param accessTokenCookieConfiguration JWT Access Token Cookie Configuration
+     * @param refreshTokenCookieConfiguration Refresh Token Cookie Configuration
+     * @param accessTokenConfiguration JWT Generator Configuration
+     * @param accessRefreshTokenGenerator Access Refresh Token Generator
+     * @param priorToLoginPersistence Prior To Login Persistence Mechanism
+     */
+    @Inject
+    public JwtCookieLoginHandler(RedirectConfiguration redirectConfiguration,
+                                 AccessTokenCookieConfiguration accessTokenCookieConfiguration,
+                                 RefreshTokenCookieConfiguration refreshTokenCookieConfiguration,
+                                 AccessTokenConfiguration accessTokenConfiguration,
+                                 AccessRefreshTokenGenerator accessRefreshTokenGenerator,
+                                 @Nullable PriorToLoginPersistence priorToLoginPersistence) {
+        super(accessTokenCookieConfiguration, redirectConfiguration, priorToLoginPersistence);
+        this.refreshTokenCookieConfiguration = refreshTokenCookieConfiguration;
+        this.accessTokenConfiguration = accessTokenConfiguration;
+        this.accessRefreshTokenGenerator = accessRefreshTokenGenerator;
+    }
+
+    private static RefreshTokenCookieConfiguration createDefaultConfiguration(JwtCookieConfiguration jwtCookieConfiguration) {
+        return new RefreshTokenCookieConfiguration() {
+            @NonNull
+            @Override
+            public String getCookieName() {
+                return "JWT_REFRESH_TOKEN";
```

Comment:
Instead of:  

`JWT_REFRESH_TOKEN`

it should be: 

`RefreshTokenCookieConfigurationProperties.DEFAULT_COOKIENAME`